src/NuGet.Core/NuGet.Configuration/Utility/SettingsUtility.cs

```diff
@@ -11,7 +11,7 @@ public static class SettingsUtility
         public const string ConfigSection = "config";
         public const string GlobalPackagesFolderKey = "globalPackagesFolder";
         public const string GlobalPackagesFolderEnvironmentKey = "NUGET_PACKAGES";
-        public const string DefaultGlobalPackagesFolderPath = ".nuget\\packages\\";
+        public const string DefaultGlobalPackagesFolderPath = ".nuget/packages/";
```

Comment:
@yishaigalatzer I did it this way to be consistent with what you do on Line 146.

Personally, I like the readonly string approach better so I've changed it to that.